pkg/cluster/service.go

```diff
@@ -25,9 +25,10 @@ const (
 	// labelClusterRolePrefix is the prefix that defines the cluster role as label key
 	labelClusterRolePrefix = "cluster-role"
 
-	toolchainAPIQPS   = 20.0
-	toolchainAPIBurst = 30
-	toolchainTokenKey = "token"
+	// These are not used
+	// toolchainAPIQPS   = 20.0
+	// toolchainAPIBurst = 30
+	// toolchainTokenKey = "token"
```

Comment:
let's maybe remove those if not used
```suggestion
```

Reply:
Yeah... I only stumbled upon these because the linter complained about them, so I was a little bit hesitant to remove unrelated code that I know nothing about... But you're right, we should get rid of those...

Reply:
+1 to removing them